pkg/skaffold/runner/label/label.go

```diff
@@ -86,67 +90,81 @@ func updateRuntimeObject(client clientgo.Interface, labels map[string]string, re
 	}
 	var err error
 	obj := *res.Obj
+	originalJSON, _ := json.Marshal(obj)
+	modifiedObj := obj.DeepCopyObject()
 	switch obj.(type) {
 	case *corev1.Pod:
-		p := obj.(*corev1.Pod)
-		addSkaffoldLabels(labels, &p.ObjectMeta)
-		pods := client.CoreV1().Pods(retrieveNamespace(res.Namespace, p.ObjectMeta))
-		_, err = pods.UpdateStatus(p)
+		apiObject := modifiedObj.(*corev1.Pod)
+		addSkaffoldLabels(labels, &apiObject.ObjectMeta)
```

Comment:
can we just access the object meta directly? Even if its just assuming that its in the untyped yaml at root -> metadata